source/Esf.Website/App/shared/layout/horizontalSplitView/horizontalSplitView.component.html

```diff
@@ -0,0 +1,12 @@
+﻿<div fxLayout="column" fxFlex (mousemove)="onMouseMove($event)" (mouseup)="onMouseUp($event)" (mouseleave)="onMouseLeave($event)" ondragstart="return false;" ondrop="return false;">
+    <div fxLayout="row" [fxFlex]="view1Height">
```

Comment:
we can improve the names of view1Height and view2Height with topViewHeight and bottomViewHeight